sebastian-rueda/tarea-4/script.js

```diff
@@ -0,0 +1,138 @@
+class Calculator
+{
+    constructor(previousOperandTextElement, currentOperandTextElement)
+    {
+        this.previousOperandTextElement = previousOperandTextElement
```

Comment:
@SebastianRuedaGlobant , all these variables should be defined as Class fields at the top of the class